### ISSUE TITLE
Add new labels for Issues and PRs

### ISSUE THREAD

**Issue by pluseg** [deployphp/deployer#1059]
| Q                 | A
| ----------------- | ---
| Issue Type        | Feature Request
| Deployer Version  | N/A
| Local Machine OS  | N/A
| Remote Machine OS | N/A

### Description 
Now a lot of issues and PRs is hanging on because of core team postpones them to v5 or just wants to rewrite them. Let's add label `Core Team` for such issues/PRs and also specify correct milestone with version. I hope that it eases contributing from community and understanding feature statuses for users.

Time estimation for setting all that label: 20 minutes.

**Comment by elfet** [comment]
Adding milestone is good think. But what do you mean by Core Team label?

**Comment by pluseg** [comment]
@elfet It's a task like 
https://github.com/deployphp/deployer/issues/1023 — has no sense as will be reworked in v5
https://github.com/deployphp/deployer/issues/1018 — as I understand you're working on it
https://github.com/deployphp/deployer/issues/968 — as todo things for future, not a task
https://github.com/deployphp/deployer/issues/846 — requires your decision about feature's usefulness
https://github.com/deployphp/deployer/issues/845 — I don't understand clearly will you do it in mentioned feature or not.
https://github.com/deployphp/deployer/pull/1043 — It's done, but you want to rework it.
https://github.com/deployphp/deployer/pull/1035 — same
https://github.com/deployphp/deployer/pull/1016 — requires solution from you as maintainer
https://github.com/deployphp/deployer/pull/866 — requires refactoring
Maybe others.

The main ideas:
1. Need to mark issues/PRs which requires decision of core team members (you and oanhnn) or already in yours `in progress` pool.
2. Need to understand which bugs/features are available for contributers (like me).
3. Need to understand what feature goes which version.
4. Merge related tasks into one big task with full description. For example all tasks blocked by future v5 ssh connections mechanism should be merged into one task with title "v5 ssh features". So we can close hanging tasks and leave only one with clear todo list.

**Comment by elfet** [comment]
Agree with you what we need to clear structure for it. What about adding milestone and assignees?
I'll add me where i'm working.

**Comment by pluseg** [comment]
@elfet cool, thanks. I think about CoreTeam label as assigners aren't displayed on https://github.com/deployphp/deployer/issues and https://github.com/deployphp/deployer/pulls pages therefore I have to open and check each ticket one by one.

**Comment by elfet** [comment]
Hmmm, well, we can try some think like CoreTeam or In progress. interesting what other open source communities do for this.

**Comment by pluseg** [comment]
@elfet I've checked Symfony project. They use:
1. Bug/Feature label
2. Label with component name: Process, Form, etc
3. `Status: Needs Review` and `Status: Waiting feedback`
4. `Projects` section of github — https://github.com/deployphp/deployer/projects

**Comment by elfet** [comment]
Added myself to issues and pr where i'm working on. Now you can see it in list.
